openapi/crmaas/crmass.json

```diff
@@ -0,0 +1,243 @@
+{
+  "openapi": "3.1.0",
+  "x-stoplight": {
+    "id": "iyzxipdhj3szx"
+  },
+  "info": {
+    "title": "crmaas",
+    "version": "1.0"
+  },
+  "servers": [
+    {
+      "url": "https://prod.apigateway.co/crm",
+      "description": "Production"
+    },
+    {
+      "url": "https://demo.apigateway.co/crm",
+      "description": "Demo"
+    },
+    {
+      "url": "http://localhost:11001/crm",
+      "description": "Local"
+    },
+    {
+      "url": "{local}/crm",
+      "description": "Local"
+    }
+  ],
+  "paths": {
+    "/{namespace}/{resourceTypeCode}/meta/fields": {
+      "parameters": [
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "namespace",
+          "in": "path",
+          "required": true,
+          "description": "The namespace, PID/AGID"
+        },
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "resourceTypeCode",
+          "in": "path",
+          "required": true,
+          "description": "The resourceTypeCode, here Contact"
+        }
+      ],
+      "get": {
+        "summary": "Get field schema",
+        "tags": [],
+        "responses": {
+          "200": {
+            "description": "Schema found",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "id": {
+                      "type": "string"
+                    },
+                    "externalId": {
+                      "type": "string"
+                    },
```

Comment:
I don't think that we should show the internal id to users. It is usually a generated UUID.
By only using the external ids the API call to update a field will be the exact same in demo and prod.